test/e2e/network_test.go

```diff
@@ -70,6 +70,17 @@ func testEastWestNetworking(t *testing.T) {
 		err = testCtx.waitUntilJobSucceeds(linuxCurlerJob.Name)
 		assert.NoError(t, err, "could not curl the Windows server from a linux container")
 
+		//Create a ClusterIP service
```

Comment:
Missing space after `//`

I suggest that we separate this out into sub-tests for each service or even make it [table driven](https://github.com/golang/go/wiki/TableDrivenTests).

Reply:
Put in the space. Do you want me to work on separating the tests here or do you guys want to add it to your next sprint?

Reply:
That is up to you. If you don't have cycles, someone can take over this PR from you during the next sprint.